statefulset-runner/api/v1/pod_webhook.go

```diff
@@ -72,7 +72,7 @@ func (r *STSPodDefaulter) Default(ctx context.Context, obj runtime.Object) error
 			cfInstanceVar := corev1.EnvVar{Name: controllers.EnvCFInstanceIndex, Value: index}
 			container.Env = append(container.Env, cfInstanceVar)
 
-			podlog.V(1).Info(fmt.Sprintf("patching-instance-index env-var - %s: %s", controllers.EnvCFInstanceIndex, index))
+			podlog.V(1).Info("patching instance index env var", controllers.EnvCFInstanceIndex, index)
```

Comment:
`Info` should get key-value pairs as additional arguments, here it takes just two values, maybe adding value keys would look better in the logs?

Reply:
My intention was that "CF_INSTANCE_INDEX" would be the key, and its value was the value. This would result in logs such as:

{ "severity": "INFO", ..., "message": "patching instance index env var", "CF_INSTANCE_INDEX": 1 }

instead of:

{ "severity": "INFO", ..., "message": "patching instance index env var", "envVarName": "CF_INSTANCE_INDEX", "envVarValue": 1 }

Reply:
sounds reasonable